docs/server/source/production-deployment-template/node-config-map-and-secrets.rst

```diff
@@ -123,6 +123,13 @@ receive requests from NGINX instance.
 It is set to ``80`` by default.
 
 
+vars.bigchaindb-wsserver-advertised-scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``bigchaindb-wsserver-advertised-scheme`` is the protocol used to access
+the WebSocket API in BigchainDB. This can be set to 'wss' (default) or 'ws'.
```

Comment:
The default is ws, no? In the BigchainDB config file (JSON), you'd put it in double quotes, e.g. "ws" or "wss"